src/go/rpk/pkg/cli/cluster/logdirs.go

```diff
@@ -81,9 +81,13 @@ where revision is a Redpanda internal concept.
 			if len(topics) > 0 {
 				listed, err := adm.ListTopics(context.Background(), topics...)
 				out.MaybeDie(err, "unable to describe topics: %v", err)
+				var exit bool
 				listed.EachError(func(d kadm.TopicDetail) {
 					fmt.Fprintf(os.Stderr, "unable to discover the partitions on topic %q: %v\n", d.Topic, d.Err)
 				})
```

Comment:
Shouldn't we add the `exit = true` below L85?
```suggestion
				listed.EachError(func(d kadm.TopicDetail) {
					fmt.Fprintf(os.Stderr, "unable to discover the partitions on topic %q: %v\n", d.Topic, d.Err)
					exit = true
				})
```

Reply:
yep absolutely, good check